### ISSUE TITLE
401 Unauthorized for example when targeting Docker Hub

### ISSUE THREAD

**Issue by bjornmagnusson** [GoogleContainerTools/jib#564]
**Description of the issue**:

401 Unauthorized towards Docker Hub, when trying helloworld example. Local docker daemon is authenticated with Docker Hub.

**Expected behavior**:
Docker image successfully built

**Steps to reproduce**: 
In example, `gradle jib --image=<docker-hub-username/helloworld>`

**Environment**: 

------------------------------------------------------------
Gradle 4.8.1
------------------------------------------------------------

Build time:   2018-06-21 07:53:06 UTC
Revision:     0abdea078047b12df42e7750ccba34d69b516a22

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.11 compiled on March 23 2018
JVM:          1.8.0_131 (Oracle Corporation 25.131-b11)
OS:           Mac OS X 10.13.2 x86_64
**`jib-maven-plugin` Configuration**: 
```xml
PASTE YOUR pom.xml CONFIGURATION HERE
```

**`jib-gradle-plugin` Configuration**: 
```groovy
PASTE YOUR build.gradle CONFIGURATION HERE
```

**Log output**: 
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more

**Additional Information**:

**Comment by coollog** [comment]
Hi @bjornmagnusson, are you using the latest version of the helloworld example? Make sure the version of Jib is 0.9.4

**Comment by bjornmagnusson** [comment]
Yes, latest one. Just cloned the repository.
Jib is 0.9.4

**Comment by coollog** [comment]
Which method did you use to set up your credentials for authenticating with your DockerHub repository?
(https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#authentication-methods)

**Comment by bjornmagnusson** [comment]
>  If you do not define credentials explicitly, Jib will try to use credentials defined in your Docker config or infer common credential helpers.

Relying on docker config

**Comment by coollog** [comment]
@bjornmagnusson Are you able to `docker push` to `bjornmagnusson/helloworld-jib` okay?
Also, do you happen to see a message like `Using credentials from Docker config for...` if you run `gradle jib --info`?

**Comment by bjornmagnusson** [comment]
Log output:
Does not look like the registry auth lookup works correctly

`gradle jib --image=bjornmagnusson/helloworld-jib --info
Initialized native services in: /Users/bjornmagnusson/.gradle/native
The client will now receive all logging from the daemon (pid: 92219). The daemon log file: /Users/bjornmagnusson/.gradle/daemon/4.8.1/daemon-92219.out.log
Starting 12th build in daemon [uptime: 2 hrs 2 mins 59.678 secs, performance: 99%, no major garbage collections]
Using 4 worker leases.
Starting Build
Settings evaluated using settings file '/Users/bjornmagnusson/Documents/git/jib/examples/helloworld/settings.gradle'.
Projects loaded. Root project using build file '/Users/bjornmagnusson/Documents/git/jib/examples/helloworld/build.gradle'.
Included projects: [root project 'helloworld']

> Configure project :
Evaluating root project 'helloworld' using build file '/Users/bjornmagnusson/Documents/git/jib/examples/helloworld/build.gradle'.
All projects evaluated.
Selected primary task 'jib' from project :
Tasks to be executed: [task ':compileJava', task ':processResources', task ':classes', task ':jib']
:compileJava (Thread[Task worker for ':',5,main]) started.

> Task :compileJava UP-TO-DATE
Skipping task ':compileJava' as it is up-to-date.
:compileJava (Thread[Task worker for ':',5,main]) completed. Took 0.023 secs.
:processResources (Thread[Task worker for ':',5,main]) started.

> Task :processResources UP-TO-DATE
Skipping task ':processResources' as it is up-to-date.
:processResources (Thread[Task worker for ':',5,main]) completed. Took 0.003 secs.
:classes (Thread[Task worker for ':',5,main]) started.

> Task :classes UP-TO-DATE
Skipping task ':classes' as it has no actions.
:classes (Thread[Task worker for ':',5,main]) completed. Took 0.0 secs.
:jib (Thread[Task worker for ':',5,main]) started.
Searching for main class... Add a 'mainClass' configuration to 'jib' to improve build speed.
warning: Base image 'gcr.io/distroless/java' does not use a specific image digest - build may not be reproducible

Containerizing application to bjornmagnusson/helloworld-jib...
Containerizing application with the following files:
        Classes:
                /Users/bjornmagnusson/Documents/git/jib/examples/helloworld/build/classes/java/main/example
        Resources:
                /Users/bjornmagnusson/Documents/git/jib/examples/helloworld/build/resources/main/world
        Dependencies:
                /Users/bjornmagnusson/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/1.3.9/40719ea6961c0cb6afaeb6a921eaa1f6afd4cfdf/jsr305-1.3.9.jar
                /Users/bjornmagnusson/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.1.3/39b109f2cd352b2d71b52a3b5a1a9850e1dc304b/error_prone_annotations-2.1.3.jar
                /Users/bjornmagnusson/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/23.6-jre/c0b638df79e7b2e1ed98f8d68ac62538a715ab1d/guava-23.6-jre.jar
                /Users/bjornmagnusson/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.1/ed28ded51a8b1c6b112568def5f4b455e6809019/j2objc-annotations-1.1.jar
                /Users/bjornmagnusson/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-compat-qual/2.0.0/fc89b03860d11d6213d0154a62bcd1c2f69b9efa/checker-compat-qual-2.0.0.jar
                /Users/bjornmagnusson/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.14/775b7e22fb10026eed3f86e8dc556dfafe35f2d5/animal-sniffer-annotations-1.14.jar

RUNNING Building and pushing image
Retrieving registry credentials for registry.hub.docker.com...
RUNNING Retrieving registry credentials for registry.hub.docker.com
Getting base image gcr.io/distroless/java...
RUNNING Pulling base image manifest
RUNNING Building application layers
Building dependencies layer...
RUNNING Building dependencies layer
Building resources layer...
RUNNING Building resources layer
Building dependencies layer : 0.958 ms
Building resources layer : 0.367 ms
No credentials could be retrieved for registry registry.hub.docker.com
Retrieving registry credentials for registry.hub.docker.com : 1.876 ms
RUNNING Authenticating with push to registry.hub.docker.com
Building classes layer...
RUNNING Building classes layer
Building classes layer : 0.827 ms
Building application layers : 3.137 ms
RUNNING Setting up to push layers
Setting up to push layers : 1.424 ms
Pulling base image manifest : 907.119 ms
RUNNING Setting up base image caching
RUNNING Pulling base image layer sha256:57752e7f9593cbfb7101af994b136a369ecc8174332866622db32a264f3fbefd
Setting up base image caching : 0.159 ms
Pulling base image layer sha256:57752e7f9593cbfb7101af994b136a369ecc8174332866622db32a264f3fbefd : 0.066 ms
RUNNING Pulling base image layer sha256:ba7c544469e514f1a9a4dec59ab640540d50992b288adbb34a1a63c45bf19a24
Pulling base image layer sha256:ba7c544469e514f1a9a4dec59ab640540d50992b288adbb34a1a63c45bf19a24 : 0.049 ms
RUNNING Pulling base image layer sha256:079855b1a2fb11564a2bc8ea9f278bcb2bfa9a7a8edbc22653fd4e2afdf5a0c4
RUNNING Setting up to push layers
Setting up to push layers : 0.661 ms
Authenticating with push to registry.hub.docker.com : 1532.793 ms
RUNNING Pushing BLOB digest: sha256:7b1e394da5158d765cec4b8ce6ca45bb0be8e8b0922510dcf04030e986c3dc6e, size: 2476724
RUNNING Pushing BLOB digest: sha256:b5b5c7d33b898d4a1075ab166afc78607af99380e3b85705480c3940f51148e7, size: 829
RUNNING Pushing BLOB digest: sha256:12590341691b549b4c86dbbe4fdccc66b0fa39104048252384ab5af1ed89d773, size: 104
RUNNING Pushing BLOB digest: sha256:57752e7f9593cbfb7101af994b136a369ecc8174332866622db32a264f3fbefd, size: 7695860
RUNNING Pushing BLOB digest: sha256:ba7c544469e514f1a9a4dec59ab640540d50992b288adbb34a1a63c45bf19a24, size: 622796
Pushing BLOB digest: sha256:57752e7f9593cbfb7101af994b136a369ecc8174332866622db32a264f3fbefd, size: 7695860 : 381.043 ms
Pushing BLOB digest: sha256:b5b5c7d33b898d4a1075ab166afc78607af99380e3b85705480c3940f51148e7, size: 829 : 383.325 ms
Pushing BLOB digest: sha256:7b1e394da5158d765cec4b8ce6ca45bb0be8e8b0922510dcf04030e986c3dc6e, size: 2476724 : 420.471 ms
Pushing BLOB digest: sha256:12590341691b549b4c86dbbe4fdccc66b0fa39104048252384ab5af1ed89d773, size: 104 : 420.817 ms
Pushing BLOB digest: sha256:ba7c544469e514f1a9a4dec59ab640540d50992b288adbb34a1a63c45bf19a24, size: 622796 : 429.62 ms

> Task :jib
Task ':jib' is not up-to-date because:
  Task has not declared any outputs despite executing actions.

Pulling base image layer sha256:079855b1a2fb11564a2bc8ea9f278bcb2bfa9a7a8edbc22653fd4e2afdf5a0c4 : 4374.607 ms
RUNNING Building container configuration
RUNNING Pushing BLOB digest: sha256:079855b1a2fb11564a2bc8ea9f278bcb2bfa9a7a8edbc22653fd4e2afdf5a0c4, size: 38708273
Building container configuration : 0.153 ms
Finalizing...
RUNNING Pushing container configuration
Pushing container configuration : 0.574 ms
RUNNING Pushing BLOB digest: sha256:8b04acb1e0ab47670054379eb36f010ba6bbb0e1dcf1f1b2527e9ca8027130a0, size: 694
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more
Got more than one input Future failure. Logging failures after the first
com.google.cloud.tools.jib.registry.RegistryUnauthorizedException: Unauthorized for registry.hub.docker.com/bjornmagnusson/helloworld-jib
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:200)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:338)
        at com.google.cloud.tools.jib.registry.RegistryClient.checkBlob(RegistryClient.java:249)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:80)
        at com.google.cloud.tools.jib.builder.steps.PushBlobStep.call(PushBlobStep.java:35)
        at com.google.common.util.concurrent.CombinedFuture$CallableInterruptibleTask.runInterruptibly(CombinedFuture.java:181)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.http.HttpResponseException: 401 Unauthorized
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1070)
        at com.google.cloud.tools.jib.http.Connection.send(Connection.java:124)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:169)
        ... 10 more
`

**Comment by bjornmagnusson** [comment]
Docker config looks like in #586

**Comment by coollog** [comment]
Okay, great, we will push a fix for #586 and let you know when the new version is released.

**Comment by coollog** [comment]
Hi @bjornmagnusson , we have released a fix in version `0.9.7` - can you retry with that new version? Closing this but feel free to reopen if the issue persists.
